server/services/diskUsageService.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * This service is not per rtorrent session, which is why it does not inherit
+ * `BaseService` nor have any use of the per user API ie. `getSerivce()`
+ */
+const EventEmitter = require('events');
+const util = require('util');
+const exec = util.promisify(require('child_process').exec);
+const diskUsageServiceEvents = require('../constants/diskUsageServiceEvents');
+
+const diskUsage = {
+  linux: () =>
+    exec('df --block-size=1 --type=ext4 --portability | tail -n+2').then(({stdout}) =>
```

Comment:
This may not include docker partitions which may be of type `overlay` (seen with `df -T`). I guess I could include those by `grep`-ing with a regex (eg. `grep -E 'ext[34]|overlay'` rather than `df --type=ext4`. Are there other funny fs types on linux to include? I guess linux could also mount ntfs drives too hmm...... :)

Reply:
This command seems to be targeting very specific volumes, can we make it more robust?

Reply:
Here are some filtering criterias used in the Prometheus. Might come in handy.
https://github.com/prometheus/node_exporter/blob/master/collector/filesystem_linux.go#L32